### PR TITLE
create EndBoss class < feature

### PR DESCRIPTION
**Pull request by SunweiWang**

### CODE REVIEW

**Comment by Michieldoesburg** [comment]
Looks good. Two things though:

Should the boss really have random speed and size? Dont we want to make it a fixed thing?

And as a sidenote, for stuff like this:

fishImage = new Image(rightImageFileName, fishImage.getWidth()
          * imgSizeMultiplier, fishImage.getHeight() * imgSizeMultiplier, true,
          true);

Can you just split it into multiple lines, because the random enters are making my eyes bleed.

**Comment by SunweiWang** [comment]
Does it look like better now? :P. yeah, you are right, fixed speed and size for end boss is probably better. 
< review

File: src/main/EndBoss.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ * If the PlayerFish collides with end boss without wielding the lance, 
+ * the PlayerFish dies. 
+ * If the PlayerFish collides with end boss with the lance, 
+ * end boss dies (it might take several hits).
+ * 
+ * @author Clinton Cao, Michiel Doesburg, Matthijs Halvemaan, Dmitry Malarev, Sunwei Wang.
+ */
+public class EndBoss extends Entity implements EnemyFishInterface {
+  private static String leftImageFileName = "SharkelliLeft.png";
+  private static String rightImageFileName = "SharekelliRight.png";
```
Comment (ClintonCao) [inline src/main/EndBoss.java]: You are using the wrong file name, this may cause an exception to be thrown! <review

**Comment by ClintonCao** [comment]
The new class looks good to me, but have you tried initializing the EndBoss? I tried to create an EndBoss, but ended up getting an exception. <feature, bug

**Comment by SunweiWang** [comment]
Well, I think it's ready to merge now, does anyone still want to have a look?

**Comment by ClintonCao** [comment]
As soon a possible please, so I can  fix the conflicts for my pull request. <feature

**Pull request merged**